### PR TITLE
Move efibootmgr command to separate script so it can be run on cloned devices. 

### PR DESCRIPTION
**Pull request by umbernhard**

### CODE REVIEW

File: config/admin-functions/lockdown.sh [hunk 1]
```diff
@@ -36,13 +36,7 @@ OUTDIR="${EFIDIR}/${TARGET}"
 DEV="$(df "$OUTDIR" | tail -1 | cut -d' ' -f1)"
 part=$(cat /sys/class/block/$(basename $DEV)/partition)
```
Comment (benadida) [inline config/admin-functions/lockdown.sh:37]: gotta move these too, I think?

File: config/admin-functions/show-vx-suite-admin-menu.sh [hunk 2]
```diff
@@ -138,6 +141,9 @@ while true; do
         read -s -n 1
     ;;
     
+    setupbootentry)
+    	bash setup-boot-entry.sh
+	read -s -n 1
```
Comment (benadida) [inline config/admin-functions/show-vx-suite-admin-menu.sh:147]: missing `;;` I think?

**Review by benadida — CHANGES_REQUESTED** [review]
caught a couple of bugs by inspection, I think, but check me. Make sure this runs properly independently and as part of the calling script.

File: config/admin-functions/lockdown.sh (hunk 1 above)
Comment (umbernhard) [reply config/admin-functions/lockdown.sh:37]: ope, good call.

File: config/admin-functions/show-vx-suite-admin-menu.sh (hunk 2 above)
Comment (umbernhard) [reply config/admin-functions/show-vx-suite-admin-menu.sh:147]: also good call.

**Review by benadida — APPROVED** [review]

**Comment by benadida** [comment]
testing this on a real machine, and seeing a couple of issues:

- the lockdown script fails on first run... but error message disappears quickly so I don't have any more info yet, I'll see what I can find.
- the setup-boot-entry script doesn't work because it doesn't have the executable bit set.

**Comment by benadida** [comment]
I tried ctrl-c'ing out of the menu and doing `sudo ./lockdown.sh` at the prompt as `vx-admin`, and that worked on the first try. Maybe there's something wrong when it's run from the menu? I'll keep trying for a more helpful repro.

**Pull request merged**